modules/manage/pages/revisions.adoc

```diff
@@ -83,101 +99,38 @@ TIP: Use the Admin Rest API endpoint for {rest-api-admin--xref--database-configu
+Sync Gateway 4.0+ uses a simplified pruning approach for xref:server-compatibility:server-compatibility-xdcr-mobile.adoc#version-vectors[version vectors]:
 
-The algorithm allows the branch to retain a configurable number of revisions (revs_limit) and removes all older revisions.
+* The number of sources preserved is at least 5, and revisions older than xref:server:learn:clusters-and-availability/xdcr-enable-crossclusterversioning.adoc#version-pruning-window-hrs[versionPruningWindowHrs] are pruned on document modification
+* 1 revision exists per client that wrote a document or resolved a conflict
+
+This approach eliminates the complexity of revision tree management while ensuring sufficient history for conflict resolution.
 
 [#lbl-rtctrl]
 === Controls
-You can vary the number of retained revisions using the Configuration File's
-{configuration-schema-database--xref--revs-limit}
-setting.
-
-The algorithm keeps the configured number of revisions (`revs_limit`) in the shortest non-tombstoned branch and removes all older revisions from that branch.
-For applications using Couchbase Lite 2.x+ with automatic conflict resolution, low `revs_limit` values, even as low as 1, provide optimal performance and storage efficiency.
-
-[NOTE]
---
-Do not set `revs_limit` below 100 when `allow_conflicts` = true
-This setting may affect the conflict resolution process, as insufficient revision history may prevent resolving conflicts.
---
-
-The default and minimum values of `revs_limit` are dependent on whether
-xref:configuration:configuration-schema-database.html#database-allow_conflicts[allow conflicts]
- is set True or False -- see <<tbl_min_default_vals>>.
-
-The process to remove obsolete revisions is called pruning and runs automatically every time a revision is added.
-Although fundamentally the same, the pruning algorithm works slightly differently between Sync Gateway and Couchbase Lite.
-On Sync Gateway, the pruning algorithm is applied to the shortest, non-tombstoned branch in the revision tree.
 
-If there are conflicting revisions, the document may end up with **disconnected branches** after the pruning process.
+You can configure revision retention using the Configuration File's {configuration-schema-database--xref--revs-limit} setting.
 
-In the animation below, the document has a conflicting branch.
-When the shortest branch reaches the `revs_limit` threshold, pruning cuts it off.
-The revision tree is not in a corrupted state and the logic that chooses the winning revision still applies.
-This prevents certain merge operations (`n-way` merge) from resolving conflicts and reduces disk space that conflict resolution could free.
+For Sync Gateway 4.0+ with automatic conflict resolution enabled, low `revs_limit` values provide optimal performance and storage efficiency.
+The Last Write Wins conflict resolution strategy supports values as low as 1.
```

Comment:
```suggestion
You can configure revision tree retention using the Configuration File's {configuration-schema-database--xref--revs-limit} setting. This only affects the number previous revision IDs stored in the metadata.
```

I think the language around tuning this value came from an overcorrect from `allow_conflicts=true` and it doesn't apply. This value should really never be changed.